Tools/CISupport/ews-build/steps.py

```diff
@@ -1239,12 +1228,16 @@ def validate_github(self, pr_number):
 
         repository_url = self.getProperty('repository', '')
 
-        pr_closed = self._is_pr_closed(pr_number, repository_url=repository_url) if self.verifyBugClosed else 0
+        pr_json = self.get_pr_json(pr_number, repository_url)
+        if not pr_json:
+            self._addToLog('stdio', 'Unable to fetch pull request {}.\n'.format(pr_number))
+
+        pr_closed = self._is_pr_closed(pr_json) if self.verifyBugClosed else 0
         if pr_closed == 1:
             self.skip_build('Pull request {} is already closed'.format(pr_number))
             return False
 
-        obsolete = self._is_pr_obsolete(pr_number, repository_url=repository_url) if self.verifyObsolete else 0
+        obsolete = self._is_pr_obsolete(pr_json) if self.verifyObsolete else 0
         if obsolete == 1:
             self.skip_build('Pull request {} (sha {}) is obsolete'.format(pr_number, self.getProperty('github.head.sha', '?')[:HASH_LENGTH_TO_DISPLAY]))
```

Comment:
Nit: we should be consistent in terminology, especially in UI, for `hash` vs `sha`. Maybe let's use hash everywhere.